packages/create-cloudflare/src/helpers/__tests__/args.test.ts

```diff
@@ -101,5 +101,15 @@ describe("Cli", () => {
 				"5",
 			]);
 		});
+
+		const stringArgs = [
+			"--framework",
+			"--template",
+			"--type",
+			"--existing-script",
+		];
+		test.each(stringArgs)("%s requires an argument", async (arg) => {
+			await expect(parseArgs(["my-react-project", arg])).rejects.toThrowError();
```

Comment:
@jculvey - I note that this test is now polluting our tests with console logs...

E.g. https://github.com/cloudflare/workers-sdk/actions/runs/8287230311/job/22679069151#step:8:48